pinot-core/src/main/java/org/apache/pinot/core/query/request/context/QueryContext.java

```diff
@@ -133,6 +137,8 @@ private QueryContext(String tableName, List<ExpressionContext> selectExpressions
     _queryOptions = queryOptions;
     _debugOptions = debugOptions;
     _brokerRequest = brokerRequest;
+    _gapfillType = null;
```

Comment:
Instead of setting `_gapfillType` to null, can we set it to `GAP_FILL_NONE` as that will clearly show that no gap filling will happen by default?

Reply:
GAP_FILL_NONE was my first version of implementation. @Jackie-Jiang proposed "null" option. I am fine with both solutions, but I do not want to make another change. I prefer to keep as it is.

@amrishlal Please let me know if you have strong opinion about it.

Reply:
Let's leave it as null as Jackie suggested.